internal/secrets/driver/driver.go

```diff
@@ -31,3 +31,13 @@ type Encrypter interface {
 	// Encrypt encrypts the plaintext and returns the cipher message.
 	Encrypt(ctx context.Context, plaintext []byte) ([]byte, error)
 }
+
+// Crypter defines an interface satisfied by implementing both
+// Encrypterand Decrypter. This composed interface reflects the
+// inherent tie between compatible pairs, ie to decrypt a message,
+// Decrypter must be based on the same underlying credentials as
+// the Encrypter which encrypted the message.
+type Crypter interface {
```

Comment:
If you haven't already filed an issue to follow up on the decision about whether to have a single interface or two separate ones, can you do so? It's related to but also somewhat independent to the "concrete type + driver/interface" discussion.

Reply:
filed https://github.com/google/go-cloud/issues/922